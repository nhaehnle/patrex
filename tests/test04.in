dolist(obj, a, 78 + 7, "Hi there", (4 + x) / 2, foo(x, y), bar->hi);
