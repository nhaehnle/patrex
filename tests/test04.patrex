
match $( dolist( $!(,)*|obj|, $( $!(,)*|param| )+(,)[params] ); )|call|
	erase call
	forall params
		insert call.start "do({obj}, {param});\n"
