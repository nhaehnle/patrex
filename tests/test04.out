do(obj, a);
do(obj, 78 + 7);
do(obj, "Hi there");
do(obj, (4 + x) / 2);
do(obj, foo(x, y));
do(obj, bar->hi);

